common/src/main/java/tachyon/security/PlainSaslServer.java

```diff
@@ -182,6 +191,7 @@ public void handle(Callback[] callbacks) throws IOException, UnsupportedCallback
 
       if (ac != null) {
         ac.setAuthorized(true);
+        RemoteClientUser.set(username);
```

Comment:
1, Done. 2, Good question. Thanks! We should pick authorization ID. Updated in latest patch.